### PR TITLE
Fix sql total duration

### PR DESCRIPTION
**Pull request by xacobofg**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #8550

### CODE REVIEW

File: inc/commonitilobject.class.php [hunk 1]
```diff
@@ -6711,7 +6711,6 @@ function viewEditSubitem" . $this->fields['id'] . "$rand(e, itemtype, items_id,
 
          $criteria = [
             'SELECT'   => 'actiontime',
```
Comment (cedric-anne) [inline inc/commonitilobject.class.php:6713]: May be we should directly request the SUM.

Comment (trasher) [reply inc/commonitilobject.class.php:6713]: I agree. I also agree with the distinct removal

Comment (OscarBeiro) [reply inc/commonitilobject.class.php:6713]: We thought about it but we didn't want to alter to much code to get it accepted ASAP :) 
@xacobofg is making the SUM.

File: inc/commonitilobject.class.php [hunk 2]
```diff
@@ -6710,15 +6710,14 @@ function viewEditSubitem" . $this->fields['id'] . "$rand(e, itemtype, items_id,
          $total_actiontime = 0;
 
          $criteria = [
-            'SELECT'   => 'actiontime',
-            'DISTINCT' => true,
+            'SELECT'   => ['SUM' => 'actiontime AS actiontime'],
```
Comment (trasher) [inline inc/commonitilobject.class.php:6713]: I would use a different name for AS to prevent confusion, but this is not a blocker

**Review by trasher — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**